### PR TITLE
Delete Notification at end point

### PR DESCRIPTION
**Pull request by tariqjavid**
# Description

 Delete Notification at end point :storyNo:#142

Fixes # (issue)

# How to test? postman

# Checklist

- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/controllers/notifications-controller.js [hunk 1]
```diff
@@ -16,7 +16,14 @@ const createNotification = async (body) => {
     id: insertedNotifications[0],
   };
 };
+// users delete by id
```
Comment (tpetrina) [inline src/server/api/controllers/notifications-controller.js:19]: Missing a blank line before the comment.

File: src/server/api/controllers/notifications-controller.js [hunk 2]
```diff
@@ -16,7 +16,14 @@ const createNotification = async (body) => {
...
+const deleteNotification = async (notiId) => {
```
Comment (tpetrina) [inline src/server/api/controllers/notifications-controller.js:20]: Rename to `notificationId`

File: src/server/api/routes/notifications-router.js [hunk 3]
```diff
@@ -44,5 +44,36 @@ router.post('/', (req, res) => {
         .end();
     });
 });
+/**
```
Comment (tpetrina) [inline src/server/api/routes/notifications-router.js:47]: Missing empty line before comment.

File: src/server/api/routes/notifications-router.js [hunk 4]
```diff
@@ -44,5 +44,36 @@ router.post('/', (req, res) => {
...
+ * @swagger
+ * /Notification/{ID}:
```
Comment (tpetrina) [inline src/server/api/routes/notifications-router.js:49]: Prefer lowercase `notification` in URLs.

File: src/server/api/routes/notifications-router.js [hunk 5]
```diff
@@ -44,5 +44,36 @@ router.post('/', (req, res) => {
...
+ *      - in: path
+ *        name: ID
+ *        description: ID of the Notification to delete.
+ *    responses:
+ *      200:
+ *        description: Notification deleted
+ *      5XX:
+ *        description: Unexpected error.
+ */
+
```
Comment (tpetrina) [inline src/server/api/routes/notifications-router.js:65]: Remove the empty line between comment and a method.

**Review by tpetrina — APPROVED** [review]
Small nitpicks, fix and then it's good to merge!

**Review by bytewiz — CHANGES_REQUESTED** [review]
See comments above

**Review by benna100 — APPROVED** [review]
Looks good

**Pull request merged**